docs/enabling-ha.md

```diff
@@ -88,6 +88,9 @@ spec:
   minReplicas: 1
 ```
 
+By default, the Webhook deployment is configured to block a [Cluster Autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler) from scaling down the node that's running the only replica of the deployment using the `cluster-autoscaler.kubernetes.io/safe-to-evict` annotation -- when the Webhook service is unreachable, Tekton resources can't be created, updated or deleted.
```

Comment:
[nit] The `--` confused me a bit reading this for the first time. Consider making this two sentences.

Reply:
Good call, done :)